runscript.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import unittest
+import xmlrunner
+import inspect
+from module.cliparser import build_parser
+from module.buildcap import build_capabilities
+from selenium import webdriver
+
+# exit codes
+ERROR = 3
+UNSTABLE = 2
+FAILED = 1
+PASSED = 0
+ret = PASSED
+
+parser = build_parser('RWD client for running a script');
+parser.add_argument('--path', required=True)
+
+parser.add_argument('--xml', dest='xml', action='store_true')
+parser.add_argument('--no-xml', dest='xml', action='store_false')
```

Comment:
can we add 'help' or comments to these arguments? it's not obvious especially because of generic names: 'path', 'xml', etc.